index.bs

```diff
@@ -1719,6 +1719,45 @@ that should be considered in the development of new features, notably:
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3>Extend existing manifest files rather than creating new ones</h3>
+
+Manifest files are sometimes required. However, they can also add complexity to the web platform,
+which creates developer confusion. The use of different manifests on the web should therefore be kept to a minimum.
+
+Metadata can reside inside a new manifest if that is related to a different function or the fetch timing 
+of that metadata is different. Application metadata should be added to the Web App Manifest or be an extension of it. 
+Parameters for a specific function (like Payment Handler Manifest) are a completely different type. Publication Manifest,
+using JSON-LD is another type as well.
```

Comment:
I don't think we should mention JSON-LD, see https://github.com/w3ctag/design-principles/issues/128.

Reply:
This doesn't really seem to suggest JSON-LD is the way to go; it just mentions that as one of many examples in a neutral way. (Not that I'm a huge fan of JSON-LD, just to be clear)

I'm more concerned about both of the examples listed after the application manifest recommendation - it lists two examples with no analysis or guidance for the reader to work with - so I'm not sure listing those two examples really help in any way at all.

Reply:
I'm opposed to mentioning JSON-LD in the design principles. I don't think including it is neutral at all.

Reply:
@annevk is it fine to leave the Publication Manifest and just not mention JSON-LD ?

Reply:
Removed.

Reply:
@kenchris it seems better to not include that either then.